libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/SalesforceSDKManager.h

```diff
@@ -31,14 +31,14 @@
 /**
  Block typedef for creating a custom snapshot view controller.
  */
-typedef UIViewController * __nullable (^SFSnapshotViewControllerCreationBlock)(void);
+typedef UIViewController * __nullable (^SFSnapshotViewControllerCreationBlock)(void)  NS_SWIFT_NAME(SnapshotViewCreationBlock);
 
 typedef NS_ENUM(NSUInteger, SFAppType) {
     kSFAppTypeNative,
     kSFAppTypeHybrid,
     kSFAppTypeReactNative,
     kSFAppTypeNativeSwift
-};
+} NS_SWIFT_NAME(AppType);
```

Comment:
Just to be sure, the swift code will look like:
```swift
AppType.Native
```

Reply:
@wmathurin It will be AppType.native.